src/stac_utils/google.py

```diff
@@ -58,10 +55,7 @@ def run_query(
 ) -> List[dict]:
     if not client:
         if not service_account_blob:
-            try:
-                service_account_blob = json.loads(os.environ["BQ_SERVICE_ACCOUNT"])
-            except (json.JSONDecodeError, KeyError):
-                pass
+            service_account_blob = json.loads(os.environ["BQ_SERVICE_ACCOUNT"])
```

Comment:
Feels like we should just call `get_credentials` with `BQ_SERVICE_ACCOUNT` as the service account env name here instead

Reply:
did either or because i'm not fucking with all our secrets lol